### PR TITLE
feat: add --set option for task runner variables

### PR DESCRIPTION
**Pull request by decleaver**
## Description

Addresses #146 adding the --set option to set variables when calling `uds run`

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Other (security config, docs update, etc)

## Checklist before merging

- [X] Test, docs, adr added or updated as needed
- [X] [Contributor Guide Steps](https://github.com/defenseunicorns/uds-template-capability/blob/main/CONTRIBUTING.md)(https://github.com/defenseunicorns/uds-template-capability/blob/main/CONTRIBUTING.md#submitting-a-pull-request) followed

### CODE REVIEW

**Review by mikevanhemert — DISMISSED** [review]

File: src/config/lang/lang.go [hunk 1]
```diff
@@ -65,4 +65,5 @@ const (
 
 	// uds run
 	CmdRunFlag = "Name and location of task file to run"
+	CmdPackageCreateFlagSet = "Specify package variables to set on the command line (KEY=value)"
```
Comment (UncleGedd) [inline src/config/lang/lang.go:68]:
```suggestion
CmdRunSetVarFlag = "Set a runner variable from the command line (KEY=value)"
```

File: src/cmd/run.go [hunk 2]
```diff
@@ -28,21 +31,28 @@ var runCmd = &cobra.Command{
...
 		}
 	},
 }
 
 func init() {
+	initViper()
 	rootCmd.AddCommand(runCmd)
 	runFlags := runCmd.Flags()
 	runFlags.StringVarP(&config.TaskFileLocation, "file", "f", config.TasksYAML, lang.CmdRunFlag)
+	runFlags.StringToStringVar(&config.SetVariables, "set", v.GetStringMapString(common.VPkgCreateSet), lang.CmdPackageCreateFlagSet)
```
Comment (UncleGedd) [inline src/cmd/run.go:57]:
```suggestion
runFlags.StringToStringVar(&config.SetVariables, "set", v.GetStringMapString(common.VPkgCreateSet), lang.CmdRunSetVarFlag)
```

**Review by UncleGedd — COMMENTED** [review]
Looks great! Suggested a couple minor changes

**Review by UncleGedd — APPROVED** [review]
thanks @decleaver !

**Pull request merged**